tendrl/commons/objects/node_context/__init__.py

```diff
@@ -29,9 +30,17 @@ def __init__(self, machine_id=None, node_id=None, fqdn=None,
         self._etcd_cls = _NodeContextEtcd
 
     def _get_machine_id(self):
-        cmd = cmd_utils.Command("cat /etc/machine-id")
-        out, err, rc = cmd.run()
-        return str(out)
+        try:
+            out = None 
+            with open('/etc/machine-id', 'r') as f:
+                out = f.read().strip('\n') 
+        except IOError as ex:
+            exc_type, exc_value, exc_tb = sys.exc_info()
+            traceback.print_exception(
+                exc_type, exc_value, exc_tb, file=sys.stderr)
+            sys.stderr.write(
```

Comment:
what is the point here? Ideally you should log using the Event(https://github.com/Tendrl/commons/pull/297/commits/80bc98313d23b8c9afc45bf0404ea9ef0c7d52b0#diff-e3ba3b401bfdec0d0df1d4ee946980abR34)?